impeller/entity/entity.cc

```diff
@@ -43,6 +43,8 @@ Entity::Entity(Entity&&) = default;
 
 Entity::Entity(const Entity&) = default;
 
+Entity& Entity::operator=(Entity&&) = default;
```

Comment:
What are we using this for?

Reply:
also what is this 🙃 ?

Reply:
This is the move assignment operator. I just needed it for the `deferred_entity = std::move(result.entity)` assignment.
(In general, we should always add the move assignment when adding the move constructor to prevent it from getting implicitly deleted.)